### PR TITLE
68 viewmodel for followerfollowing page

### PR DESCRIPTION
**Pull request by fdejean**
The idea is to have a friend page (follower/following UI) that is dynamic and based on the friends stored on the firebase database

### CODE REVIEW

**Comment by fdejean** [comment]
Waiting for #67 to be merge into main to put this PR in review

File: app/src/main/java/com/android/feedme/MainActivity.kt [hunk 1]
```diff
@@ -43,6 +45,7 @@ class MainActivity : ComponentActivity() {
               // Navigation host for the app
               val navController = rememberNavController()
               val navigationActions = NavigationActions(navController)
+              val profileViewModel = ProfileViewModel()
```
Comment (ohamz) [inline app/src/main/java/com/android/feedme/MainActivity.kt:48]: The instantiation has to be modified as the new way, 

Done in another PR and will be fixed when merging.

**Review by ohamz — APPROVED** [review]
I like the navigation system used, can we do this for any subscreen ? If yes I will adapt my Camera route and navigation implementation.

Otherwise, approved !

**Comment by fdejean** [comment]
> I like the navigation system used, can we do this for any subscreen ? If yes I will adapt my Camera route and navigation implementation.

We can but it has to be useful, as for example the recipe with their Id, see the Notion guide on how to implement!
